usaspending_api/awards/v2/views/idvs/activity.py

```diff
@@ -46,10 +46,7 @@
         ca.period_of_performance_start_date,
         ca.piid,
         rl.legal_business_name                          recipient_name,
-        rl.recipient_hash || case
-            when tf.ultimate_parent_unique_ide is null then '-R'
-            else '-C'
```

Comment:
Why was this logic removed? Didn't Seth have a PR that required this?

Reply:
It was moved down into the FROM clause.

Reply:
It looks like it was modified and moved below, not removed